src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+const SEX_MALE = 'm';
+
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+
+  const getPartnerInfo = () => {
+    if (isMarried) {
+      return `${partnerName} is my ${sex === SEX_MALE ? 'wife' : 'husband'}`;
+    }
+
+    return 'I am not married';
+  };
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {typeof age === 'number' && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {getPartnerInfo()}
```

Comment:
It's bad practice to use functions inside `jsx`
As it's hard to predict what function will return and what result user will see on the page
Let's avoid function here
Just prepare the content based on variables before the return statement